echopype/echodata/combine.py

```diff
@@ -273,20 +281,23 @@ def in_memory_combine(
         if group_path is None:
             group_path = "Top-level"
 
-        for echodata in echodatas:
-            if echodata[group_path] is not None:
-                group_datasets.append(echodata[group_path])
+        for ed in echodatas:
```

Comment:
Note: I renamed the variable `echodata` to the shorter `ed`, here and below, for readability. IMHO.

Reply:
I like this. I always have problems reading code using `x` and `xs` in loops... convenient but can easily be confusing.